README.md

```diff
@@ -1,70 +1,185 @@
+1. Start cloning this repository by entering the following command:
 
-### Making a Progressive Web App
+<pre><code>git clone https://github.com/adebsa2401/Wiki-Country.git</code></pre>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+2. Install npm dependencies by running <pre><code>npm i</code></pre>
 
-### Advanced Configuration
+3. Build assets by running <pre><code>npm run build</code></pre>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+4. Finally run <pre><code>npm start</code></pre> to start local node.js server. The project's entry file `index.html` should automatically open in your browser. If it doesn't, open your browser and head over to the following url: `http://localhost:3000`
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [node.js / npm](https://nodejs.org/en/download/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<pre><code>git clone https://github.com/adebsa2401/Wiki-Country.git</code></pre>
+
+### Install
+
+Install this project with:
+
+<pre><code>npm i</code></pre>
+
+### Usage
+
+To run the project, execute the following command:
+
+<pre><code>npm run build</code></pre>
+<pre><code>npm start</code></pre>
+
+### Run tests
+
+To run tests, run the following command:
+
+<pre><code>npm run test</code></pre>
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+<pre><code>git subtree push --prefix build origin gh-pages</code></pre>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ben Sales**
+
+- GitHub: [@adebsa2401](https://github.com/adebsa2401)
+- Twitter: [@Adebsa3](https://twitter.com/Adebsa3)
+- LinkedIn: [Ben Sales](https://www.linkedin.com/in/adebsa/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Save reserved missions and rockets to localStorage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+- Microverse
+
```

Comment:
- [ ] Kindly add the Acknowledgment section and include credits to Nelson Sakwa whose design was used for this project.